### ISSUE TITLE
Online Update

### ISSUE THREAD

**Issue by PengWenChen** [hwsel/LiveDeep#2]
Why the `if` condition is not `Times == 0` here?
Since the model will see the ground truth when `Times > 0`, which supposed not to happen in real-time streaming video, right?

https://github.com/hwsel/LiveDeep/blob/8d291c73ff8d1f18e8332cb1bf99c914f2dd0255/VideoProcessingPMultipleParameters.py#L529

**Comment by XianglongFeng** [comment]
Yes. The "Times" should be "0". It's updated by now.

**Comment by XianglongFeng** [comment]
Yes, you are right. It should be “Times == 0”, and we have updated the code by now. Thank you for pointing out this bug in the code.

We expect that this bug fix would cause slight changes in the experimental results without impacting the claims & conclusions made in the paper, since with one iteration of training the model could not learn much. To validate this, we collected the updated results for Figure 5 in the paper, as shown below. 
![Capture](https://user-images.githubusercontent.com/16802814/132248126-e9f98bc6-1c35-4502-9985-1b1412542a10.PNG)


We notice that, with the bug fix, the accuracy for CNN decreases no more than 8%, and the accuracy for LiveDeep decreases no more than 4%, compared to the original results reported in the paper. We plan to continue collecting the updated results for other figures/tables that may be impacted by this bug fix.
